api/_lib/script/build.ts

```diff
@@ -61,12 +56,21 @@ async function main() {
       'utf-8'
     );
     const packageJson = JSON.parse(packageJsonRaw);
-    const tarballName = `${packageJson.name
-      .replace('@', '')
-      .replace('/', '-')}-v${packageJson.version}-${sha.trim()}.tgz`;
+    const files = await fs.readdir(fullDir);
+    const tarballName = files.find(f => /^vercel-(.+)\.tgz$/.test(f));
```

Comment:
Not a big deal but you don't need the capture group.

```suggestion
    const tarballName = files.find(f => /^vercel-.+\.tgz$/.test(f));
```